beetsplug/mpdupdate.py

```diff
@@ -112,18 +75,44 @@ def __init__(self):
             if self.config[key].exists():
                 config['mpd'][key] = self.config[key].get()
 
+        self.register_listener('database_change', self.db_change)
+
+    def db_change(self, lib):
+        self.register_listener('cli_exit', self.update)
```

Comment:
That's right. I still think that in this case it is more elegant than introducing a variable `database_changed` on the plugin instance and setting it to `True` upon a call to `db_change`, and checking `database_changed` in the `update` method, which was what I initially did.
If you'd prefer it that way I'll push an adequate commit.